src/components/restartGame.js

```diff
@@ -0,0 +1,11 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME } from "../pages/texts.js";
+
+/* 다시 시작하는 기능 */
+export default async function restartGame() {
+  while (true) {
+    const userInput = await MissionUtils.Console.readLineAsync(GAME.RESTART);
+    if (userInput === "1") return true;
+    else if (userInput === "2") return false;
```

Comment:
상단 if문에서 return을 해주고 있기때문에 아래처럼 수정 가능해 보입니다!

https://github.com/airbnb/javascript#blocks--no-else-return
```suggestion
    if (userInput === "1") return true;
    if (userInput === "2") return false;
```